sats-dna/src/main/kotlin/com/sats/dna/components/upcomingworkouts/UpcomingWorkout.kt

```diff
@@ -1,14 +1,18 @@
 package com.sats.dna.components.upcomingworkouts
 
-data class ScheduledWorkout(
+import androidx.compose.ui.graphics.Color
+
+data class UpcomingWorkout(
     val id: String,
     val day: String,
     val time: String,
     val duration: String,
     val name: String,
-    val location: String,
-    val instructor: String,
+    val workoutType: String?,
+    val location: String?,
+    val instructor: String?,
     val waitingListStatus: WaitingListStatus?,
+    val workoutTypeColor: Color? = null,
```

Comment:
Should we instead make `workoutType` a `UpcomingWorkoutType?` and determine the color based on that? Or are there cases where we want a GX type with some non-GX color? 🤔

Reply:
Yeah there are I think, because WorkoutType can be the entire list from the form with like 20 different types.